src/components/PicturesCollection/PicturesCollection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+import Article from './Article/Article';
+
+import styles from './PicturesCollection.module.css';
+
+const PicturesCollection = () => {
+  return (
+    <div>
+      <Link
+        to={{
+          pathname: '/new-post',
+        }}
+        style={{ textDecoration: 'none' }}
+        className={styles.Button}
+      >
+        Dodaj nowe
+      </Link>
+      <main className={styles.PicturesCollection}>
+        {' '}
```

Comment:
Pytanie: w jakim celu to: {' '}? :)

Reply:
Wygląda na to, że niepotrzebne :) w VSC formatowanie robi mi czasem coś takiego. Usunę :)